app/views/events/_form.html.erb

```diff
@@ -1,8 +1,7 @@
-<div class="container">
-  <%= form_with(model: event, local: true) do |form| %>
-    <% if event.errors.any? %>
-      <div id="error_explanation">
-        <h2><%= pluralize(event.errors.count, "error") %> prohibited this event from being saved:</h2>
+<%= form_with(model: event, local: true) do |form| %>
+  <% if event.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(event.errors.count, t('.error')) %> <%= t('.prohibited_event') %></h2>
 
```

Comment:
You didn't move the rest of elements in this file. You should be consistent with your indentations.